@PHSOptions/PHSOptions.m

```diff
@@ -204,28 +204,3 @@
             
    end
 end
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
```

Comment:
Again, changes in this file are due to switching the inheritance from
matlab.mixin.SetGet to matlab.mixin.CustomDisplay